hpi-swt2/lecture-portal
# A lecturer can start a poll

This PR adds polls to lectures. Each lecture can now have multiple polls while only one can be active. Each poll has multiple poll options but at least 2.

Testing steps:
- log in, create a lecture
- on the lectures page create a new poll by clicking new poll
- create as my polls with as many options as you want
- all things in the acceptance criteria should be given

Acceptance Criteria:
- A question text can be specified
- Single/multiple choice can be specified without reloading the page
- Number of options can be specified without reloading the page
- Answer options don't need additional information
- If I add the wrong number of answer options I can correct it
- I can't add a new poll while a different one is active
- Starting a new poll doesn't take longer than 30 seconds

Solves issue #18.

Steps needed for migration:
- A simple `bundle exec rake db:migrate` should do the trick
